content/developer/reference/javascript/patching_code.rst

```diff
@@ -0,0 +1,194 @@
+=============
+Patching code
+=============
+
+Sometimes, we need to customize the way the UI works.  Many common needs are
+covered by some supported API. For example, all registries are good extension
+points: the field registry allow adding/removing specialized field components,
+or the main component registry allow adding components that should be displayed
+all the time.
+
+However, there are situations for which it is not sufficient. In those cases, we
+may need to modify an object or a class in place. To achieve that, Odoo
+provides the utility function `patch`. It is mostly useful to override/update
+the behaviour of some other component/piece of code that one does not control.
```

Comment:
not sure about the convention, I'd have assumed that we used us english, not british